packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-details.module.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+
+import routing from './job-details.routing';
+
+import dataProcessingJobDetailsComponent from './job-details.component';
+import jobDashboard from './job-dashboard';
+import jobLogs from './job-logs';
+
+const moduleName =
+  'ovhManagerPciProjectDataProcessingSubmitJobComponentLazyLoading';
```

Comment:
```suggestion
  'ovhManagerPciProjectDataProcessingSubmitJob';
```
I think there was a confusion between the name of the lazy loaded module and the modules